### ISSUE TITLE
Ｆｕｒｉｇａｎａ Problem

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mvysny/aedict#443]
```
When searching for a kanji and checking out the examples some furigana seem to 
have gotten corrupted in the way that their translation stays in the infinitive 
form of a verb. 

In this example screenshot you can see what I mean.

```

Original issue reported on code.google.com by `dezorian...@gmail.com` on 8 Mar 2015 at 7:04

Attachments:
* [Screenshot_2015-03-08-20-00-57.png](https://storage.googleapis.com/google-code-attachments/aedict/issue-443/comment-0/Screenshot_2015-03-08-20-00-57.png)

**Comment by mvysny** [comment]
That is because the Tatoeba data files are wrong in this case. The sentence's number is 226691, it reads    お腹がすいていませんか and appropriate B-Line is お腹が空く{お腹がすいていません} か. B-Line is a way to de-conjugate and attach readings to a kanji, in this case, the sentence form is お腹がすいていません and de-conjugated form is お腹が空く, from this form Aedict must additionally guess a reading (by a dark magic ;). The fixed B-Line should instead read something like this:
おなか{お腹} が 空く{すいて} いません か
So, in short, this is a bug in Tatoeba data files, there is nothing Aedict can do. But, you can report such broken sentences to the Tatoeba guys - just open the sentence in Aedict and click the "Show sentence on Tatoeba web page" upper-right actionbar button.
